modules/ROOT/pages/munit-maven-plugin.adoc

```diff
@@ -424,7 +424,7 @@ mvn clean test
 
 === Run a Specific MUnit Test Suite
 
-By using the property `munit.test`, you can instruct the MUnit Maven plugin to run only tests that belong to a specific test suite:
+By using the property `munit.test`, you can instruct the MUnit Maven plugin to run on certain tests that belong to a specific test suite:
```

Comment:
so it doesn't or might not run on some tests that belong to a specific test suite?

Reply:
Let's say your "Test Suit X" is composed of:
- Test 1
- Test 2
- Test 3
With the `munit.test` property, you can run:
- Test 1
- Test 3
Test 2 will not run
@egrave, please, can you confirm?
Thanks!

Reply:
that's correct

Reply:
Thanks for your contribution!